saml-handler/pom.xml

```diff
@@ -0,0 +1,378 @@
+    <version>37</version>
+    <relativePath />
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.apache.sling.auth</groupId>
+  <artifactId>saml2</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+  <name>SAML2 Service Provider</name>
+
+  <properties>
+    <opensaml.version>3.4.3</opensaml.version>
+    <sling.host>localhost</sling.host>
+    <sling.port>8080</sling.port>
+    <sling.user>admin</sling.user>
+    <sling.password>admin</sling.password>
+  </properties>
+
+  <packaging>bundle</packaging>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>3.5.0</version>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+            <Bundle-Activator>org.apache.sling.auth.saml2.Activator</Bundle-Activator>
+            <Embed-Dependency>*;scope=compile|runtime;</Embed-Dependency>
+            <Embed-Transitive>true</Embed-Transitive>
+            <Embed-Directory>target/dependency</Embed-Directory>
+            <Embed-StripGroup>true</Embed-StripGroup>
+            <Import-Package>
+              !sun.misc;resolution:=optional,
+              !com.sun.msv.*;resolution:=optional,
+              !com.sun.jdmk.comm;resolution:=optional,
+              sun.io.*;resolution:=optional,
+              com.beust.jcommander*;resolution:=optional,
+              com.google.appengine.api.*;resolution:=optional,
+              com.google.apphosting.api.*;resolution:=optional,
+              com.google.appengine.repackaged.*;resolution:=optional,
+              org.apache.log.*;resolution:=optional,
+              org.apache.oro.text.perl.*;resolution:=optional,
+              antlr.*;resolution:=optional,
+              org.apache.tools.ant.*;resolution:=optional,
+              junit.framework.*;resolution:=optional,
+              org.dom4j.*;resolution:=optional,
+              org.jdom.*;resolution:=optional,
+              org.bouncycastle.*;resolution:=optional,
+              com.sun.org.apache.xerces.internal.*;resolution:=optional,
+              *</Import-Package>
+            <_noee>true</_noee>
+          </instructions>
+        </configuration>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.sling</groupId>
+        <artifactId>maven-sling-plugin</artifactId>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>8</source>
+          <target>8</target>
+        </configuration>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>2.6</version>
+        <configuration>
+          <archive>
+            <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+            <manifestEntries>
```

Comment:
Have you considered adding these entries to the bnd.bnd or pom.xml config of the maven-bundle-plugin?

Reply:
I tried taking this suggestion using bnd-maven-plugin without maven-jar-plugin. The doc https://github.com/bndtools/bnd/tree/master/maven/bnd-maven-plugin suggest it is typical to use maven-jar-plugin. I tried removing it but could not make it result in a OSGI bundle. If this is important please provide a little bit more detail. Thanks